lib/index.ts

```diff
@@ -19,8 +19,9 @@ import { readFileSync } from 'fs'
 const minNodeVersion = (process && process.env && process.env.YARGS_MIN_NODE_VERSION)
   ? Number(process.env.YARGS_MIN_NODE_VERSION)
   : 12
-if (process && process.version) {
-  const major = Number(process.version.match(/v([^.]+)/)![1])
+const nodeVersion = process && (process.versions ? process.versions.node : process.version.slice(1))
```

Comment:
Could you do:

```
const nodeVersion = process?.versions?.node && process.version.slice(1);
```

To simplify the logic slightly?

Reply:
this should be an `||` but yes the logic could be simplified. i wasn't sure if i was able to use `?.` in the code.